src/HtmlUtils.js

```diff
@@ -102,10 +102,10 @@ var sanitizeHtmlParams = {
     // URL schemes we permit
     allowedSchemes: ['http', 'https', 'ftp', 'mailto'],
 
-    // DO NOT USE. sanitize-html allows all URL starting with '//'
-    // so this will always allow links to whatever scheme the
-    // host page is served over.
-    allowedSchemesByTag: {},
+    allowedSchemesByTag: {
+        img: [ 'data', 'mxc' ],
```

Comment:
There's not much value in just including mcx in here: this will just pass it through to the browser which won't understand it. We'd have to go & parse the HTML (or use the sanitiser's parsing) and replace the mxc URLs with HTTP ones.

Also, isn't allowing data URLs going to open us up to SVG billion laugh attacks and so forth again (without the user having to click anything).